airbyte-integrations/connectors/source-greenhouse/Dockerfile.test

```
@@ -0,0 +1,22 @@
+FROM airbyte/base-python-test:dev
+
+RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
+
+ENV CODE_PATH="integration_tests"
+ENV AIRBYTE_TEST_MODULE="integration_tests"
+ENV AIRBYTE_TEST_PATH="SourceGreenhouseStandardTest"
+
+LABEL io.airbyte.version=0.1.0
+LABEL io.airbyte.name=airbyte/source-greenhouse-standard-test
+
+WORKDIR /airbyte/integration_code
+COPY source_greenhouse source_greenhouse
+COPY $CODE_PATH $CODE_PATH
+COPY sample_files/*.json $CODE_PATH
```

needed by docker if there are multiple files being copied

```suggestion
COPY sample_files/*.json $CODE_PATH/
```